src/rocksdb.rs

```diff
@@ -276,7 +278,8 @@ impl DB {
     	if cfs.len() != cf_opts.len() {
 			return Err(format!("Mismatching number of CF options"));
 		}
-        let cpath = match CString::new(path.as_bytes()) {
+        let encoded_path = Encoding::ANSI.to_bytes(path);
+        let cpath = match CString::new(encoded_path.unwrap()) {
```

Comment:
`unwrap()` is an assertion. Favor `expect` which lets you provide a proof for why the assertion will never fail